### ISSUE TITLE
Support MLT (MapLibre Tiles)

### ISSUE THREAD

**Issue by maxammann** [georust/geozero#227]
There is a new specification for MapLibre tiles: https://github.com/maplibre/maplibre-tile-spec

I don't have details yet on what it would need to write a Rust decoder but @nyurik and @mactrem might know more.

**Comment by nyurik** [comment]
First task would be a speedypfor implementation.  Basically a deserializer for a sequence of integers. There is a c implementation, so the first pass could be a simple wrapper for that

**Comment by maxammann** [comment]
Can't find it, not this one? https://github.com/lemire/FastPFor

**Comment by nyurik** [comment]
Yep that's the one -- see https://github.com/maplibre/maplibre-tile-spec/blob/e5ed469f079910ac307f973f809e5761d1924dde/spec/ENCODINGS.md#simd-fastpfor

**Comment by jo-chemla** [comment]
In case it's useful, fast forward to today and there is now an existing implementation of a rust MLT encoder/decoder here [rust/mlt](https://github.com/maplibre/maplibre-tile-spec/tree/main/rust/mlt)!
